.github/workflows/prebuilt_docker_image.yml

```diff
@@ -52,14 +51,17 @@ jobs:
         uses: docker/metadata-action@v3
         with:
           images: ${{ env.GHCR_REGISTRY }}/${{ env.IMAGE_NAME }}
+      - name: Get docker environment version
+        id: pyodide_env
+        run: echo "::set-output name=tag::$(cat .docker-env-version | tr -d " \t\n\r")"
       - name: Build and push Docker image to ${{ env.GHCR_REGISTRY }}
         id: build
         uses: docker/build-push-action@v2
         with:
           file: ./Dockerfile-prebuilt
           push: true
           build-args: |
-            VERSION=${{ env.PYODIDE_ENV_VERSION }}
+            VERSION=${{ steps.pyodide_env.outputs.tag }}
```

Comment:
This line and the change above needs updating as well. Thanks!

Reply:
Oops, thanks!